buffer/src/test/java/io/netty/buffer/AdaptiveByteBufAllocatorTest.java

```diff
@@ -229,6 +232,69 @@ public void jfrChunkAllocation() throws ExecutionException, InterruptedException
         }
     }
 
+    @SuppressWarnings("Since15")
+    @Test
+    @EnabledForJreRange(min = JRE.JAVA_17) // RecordingStream
+    @Timeout(10)
+    public void shouldCreateTwoChunks() throws Exception {
```

Comment:
This test is the a prerequisite for `shouldReuseTheSameChunk` since the latter can pass if, for some reason, we decide to increase the number of segments per chunk